ios/Classes/BetterPlayerPlugin.m

```diff
@@ -431,6 +463,13 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
             result(@([player absolutePosition]));
         } else if ([@"seekTo" isEqualToString:call.method]) {
             [player seekTo:[argsMap[@"location"] intValue]];
+
+            // re-enable PiP and Control center buttons
+            [player setIsDisplayPipButtons:true];
+            MPRemoteCommandCenter *commandCenter = [MPRemoteCommandCenter sharedCommandCenter];
+            [commandCenter.togglePlayPauseCommand setEnabled:YES];
+            [commandCenter.playCommand setEnabled:YES];
+            [commandCenter.pauseCommand setEnabled:YES];
```

Comment:
Could you add a flag or condition for preventing these process fire multi times (like [L166](https://github.com/dwango-nfc/betterplayer/blame/cb29fa0d202e2fce2dd41b614aa56476a44ba8ef/ios/Classes/BetterPlayerPlugin.m#L166))? We should reduce the effect or cost of them.